cli/tests/integration_tests.rs

```diff
@@ -53,6 +53,34 @@ fn x_deno_warning() {
   drop(g);
 }
 
+#[test]
+fn run_from_stdin() {
+  let cat = Command::new("cat")
+    .current_dir(util::root_path())
+    .arg("cli/tests/001_hello.js")
+    .stdout(std::process::Stdio::piped())
+    .spawn()
+    .unwrap();
+  let process = util::deno_cmd()
+    .current_dir(util::root_path())
+    .arg("run")
+    .arg("-")
+    .stdout(std::process::Stdio::piped())
+    .stdin(std::process::Stdio::from(cat.stdout.unwrap()))
+    .spawn()
+    .unwrap();
+  let output = process.wait_with_output().unwrap();
+  assert!(output.status.success());
+  let mut desired_output = std::fs::read_to_string(
+    util::root_path().join("cli/tests/001_hello.js.out"),
+  )
+  .unwrap();
+  desired_output.truncate(desired_output.len() - 1);
+
+  let stdout_str = std::str::from_utf8(&output.stdout).unwrap().trim();
+  assert_eq!(desired_output, stdout_str);
```

Comment:
Rather than reading `cli/tests/001_hello.js.out` just do 
```rust
let actual = std::str::from_utf8(&output.stdout).unwrap().trim();
assert_eq!("Hello World", actual);
```

Reply:
If we hardcode the output, it might also be best to hardcode the input?

Reply:
that would be better, yes